apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2/TranslationModels.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Translation.V2
+{
+    /// <summary>
+    /// Model logic.
+    /// </summary>
+    internal static class TranslationModels
+    {
+        private const string BaseApiName = "base";
+        private const string NeuralMachineTranslationApiName = "nmt";
+
+        internal static void ValidateModel(TranslationModel model)
+        {
+            if (model == TranslationModel.Base || model == TranslationModel.NeuralMachineTranslation)
```

Comment:
nit: Might be nicer and more consistent with the rest of this class as a switch statement.